dev/com.ibm.ws.security.saml.websso.2.0/src/com/ibm/ws/security/saml/sso20/internal/utils/RequestUtil.java

```diff
@@ -160,6 +160,23 @@ public static void createCookie(HttpServletRequest req,
                                                          req);
         response.addCookie(c);
     }
+    
+    public static void createCookie(HttpServletRequest req,
+                                    HttpServletResponse response,
+                                    String cookieName,
+                                    String cookieValue, int maxAge) {
+        // cookieName and cookieValue has been verified as non-null
+        WebAppSecurityConfig webAppSecurityConfig = WebAppSecurityCollaboratorImpl.getGlobalWebAppSecurityConfig();
+        ReferrerURLCookieHandler referrerURLCookieHandler = webAppSecurityConfig.createReferrerURLCookieHandler();
+        Cookie c = referrerURLCookieHandler.createCookie(cookieName,
+                                                         cookieValue,
+                                                         req);
+        c.setMaxAge(maxAge); // seconds
```

Comment:
Could just call the variable something like `maxAgeInSeconds` to avoid the need to clarify in a comment.